dev/tidb-in-kubernetes/upgrade/tidb-cluster.md

```diff
@@ -58,3 +58,25 @@ You can change the configuration of TiDB cluster through the following steps:
 >
 > - Changing the `enableConfigMapRollout` variable against a running cluster will trigger a rolling update of PD, TiKV, TiDB servers even if there's no change to the configuration.
 > - Currently, PD's `scheduler` and `replication` configurations (the configuration key under `[scheduler]` and `[replication]` of the PD configuration file) can not be upgraded automatically.  You must upgrade them manually via `pd-ctl`. See [pd-ctl](/dev/reference/tools/pd-control.md) for reference.
+
+## Force an upgrade of TiDB cluster
+
+If the PD cluster is unavailable due to factors such as PD configuration error, PD image tag error and NodeAffinity, then [scaling the TiDB cluster](/dev/tidb-in-kubernetes/scale-in-kubernetes.md), [upgrading the TiDB cluster](#upgrade-the-version-of-tidb-cluster) and [changing the TiDB cluster configuration](#change-the-configuration-of-tidb-cluster) cannot be operated. In this case, you can use `force-upgrade` (the version of TiDB Operator must be later than v1.0.0-beta.3) to force an upgrade of the cluster to recover cluster functionality. The specific steps are as follows:
```

Comment:
"PD image tag error and NodeAffinity, etc."

Reply:
Thanks for pointing out. I rephrased it by using "such as", which is more suitable here. Is there anything else?

Reply:
No, thanks!